tensorflow/core/kernels/mkl/mkl_dequantize_op.cc

```diff
@@ -78,49 +75,48 @@ class MklDequantizeOp : public OpKernel {
       MklDnnThreadPool eigen_tp(ctx);
       reorder_stream.reset(CreateStream(&eigen_tp, cpu_engine));
 
+      memory::format_tag dst_layout_type;
+      switch (src_tf_shape.dims()) {
+        case 1:
+          dst_layout_type = memory::format_tag::x;
+          break;
+        case 2:
+          dst_layout_type = memory::format_tag::nc;
+          break;
+        case 3:
+          dst_layout_type = memory::format_tag::tnc;
+          break;
+        case 4:
+          dst_layout_type = memory::format_tag::nhwc;
+          break;
+        case 5:
+          dst_layout_type = memory::format_tag::ndhwc;
+          break;
+        default:
+          OP_REQUIRES_OK(ctx,
+                         errors::Aborted("Input dims must be <= 5 and >= 1"));
```

Comment:
Nit: I think we can use invalid argument.
```suggestion
                         errors::InvalidArgument("Input dims must be <= 5 and >= 1"));
```